ln-simln-jamming/src/parsing.rs

```diff
@@ -128,6 +128,14 @@ pub struct Cli {
     /// The alias of the attacking node.
     #[arg(long)]
     pub attacker_alias: String,
+
+    /// Use incoming reputation for the simulation.
+    #[arg(long)]
+    pub incoming_reputation: bool,
+
+    /// Use outgoing reputation for the simulation.
+    #[arg(long)]
+    pub outgoing_reputation: bool,
```

Comment:
`incoming/outgoing_reputation_only` to make a it a little clearer that we're restricting what we check?
could also be interpreted as needing both set for bidirectional, but actually you should set neither.